package.json

```diff
@@ -858,7 +858,7 @@
     "coverage-all": "TEST_LIGHTSPEED_URL='http://127.0.0.1:3000' yarn _coverage-all",
     "als-compile": "yarn workspace @ansible/ansible-language-server compile"
   },
-  "version": "2.13.0",
+  "version": "24.4.0",
```

Comment:
This format sounds wrong to me, "24.4.0", for instance "24.2.0" > "24.12.0". It would be better to use 20240403 instead.

Reply:
Sorry, I'm wrong:

```
$ rpmdev-vercmp 25.02 25.12
25.2 < 25.12
```